es-ds-styles/scss/_reviews.scss

```diff
@@ -0,0 +1,208 @@
+@use "../../variables";
```

Comment:
Do we need this file?

1. I think the contents of this file are already included in `es-ds-styles/scss/vue/components/_reviews.scss`
2. In order for the contents of this new file to be included in the SASS bundle, you'd need to reference it from `bootstrap.scss`

I think we can safely delete this one, since styles for this component are already covered by that other SCSS file.